scripts/generate.sh

```diff
@@ -46,6 +46,6 @@ done
 
 jsonnet -S -c -m "${GEN_DIR}" vendor/generator/dependencies.libsonnet
 
-jrsonnet -J vendor \
+jsonnet -J vendor \
```

Comment:
Can you share the error? I don't see an issue on my machine :tm:

Reply:
Yes sure, I should've included it already in the intro, sorry for that!

```
+ jrsonnet -J vendor -S -c -m /<path-to-repo>/grafonnet/gen/grafonnet-v10.2.0/docs/ --exec '(import '\''doc-util/main.libsonnet'\'').render(import '\''/<path-to-repo>/grafonnet/gen/grafonnet-v10.2.0/main.libsonnet'\'')'
TRACE: render.libsonnet:18 warning: annotation both defined as object and package
no such field: any
                                                    : field <any> access
    vendor/doc-util/./render.libsonnet:106:15-39    : function <hasFields> call
    vendor/doc-util/./render.libsonnet:106:19-106:20: if condition
                                                    : evaluating argument
    vendor/doc-util/./render.libsonnet:93:8-93:9    : function <std.join> call
    vendor/doc-util/./render.libsonnet:89:22-38     : function <toString> call
```

Reply:
I assume it is using [this jrsonnet](https://github.com/CertainLach/jrsonnet)?

```
❯ jrsonnet --version
jrsonnet 0.4.2
```

Reply:
What do you think about reverting the commit for this, and I turn it into a separate issue? As long as the output is correct and the same for both, it probably isn't relevant to figure this out for v10.2.0?

Reply:
Yes, that jrsonnet, the master branch has received quite a bit of updates since that version, I'm using a newer version, see pre-releases here: https://github.com/CertainLach/jrsonnet/releases

Reply:
If you can drop the `jrsonnet` changes in these scripts, then I think we can merge this PR.

Reply:
Done!